### DISCUSSION TITLE
#40 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Riely007** [tog-ETH-er/togETHer#16] [General]
Agenda:

1. Community updates
2. Fireside discussion

> 会议主题：togETHer
> 会议时间： 21:00-22:00
> 重复周期：每两周 (周一)
> 
> 点击链接入会，或添加至会议列表：
> https://meeting.tencent.com/s/LxNajKWJw4RV
> 
> 会议 ID：891 5098 6775

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 思考如何提高翻译质量

**Comment by Thegaram** [comment]
### Crypto Tech Night

- 2022.05.19: CTN#14 Layer2 https://www.bilibili.com/video/BV1rU4y1m7vw
- 2022.06.02: CTN 闲聊#2 硬件钱包 with 刘力心

**Comment by winkybupt** [comment]
## 原语里弄
1. 协助ETHShanghai大会
2. 策划「领读活动」(https://primitiveslane.notion.site/8d8bb573578047a9a779e3114d4c8e7c)
3. 筹备两周年活动

**Comment by stephanieeee-ho** [comment]
### ECN
1. 常规的翻译和周报
2. rocket pool 的活动
————————
抱歉有事，无法参与此次会议

**Comment by bobjiang** [comment]
- share "GitcoinDAO decentralization journey" in SeeDAO
- https://gov.gitcoin.co/t/improve-gtc-liquidity-with-tokemak/10725/3
- GitcoinDAO purpose https://gov.gitcoin.co/t/proposal-community-input-on-gitcoindao-s-draft-purpose-essential-intents/10662
- Greenpill distributor https://gov.gitcoin.co/t/become-a-greenpill-book-distributor/10697/2
- ENSDAO grant https://discuss.ens.domains/t/agenda-for-weekly-ecosystem-meeting-may-16-2022-7pm-gmt/12683
- GR14, June 8 - 23
- Support ETHShanghai

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

22个PR #749 ～ #771

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## basic task translate

开始针对前29个基础任务的README进行中转英翻译，已经召集6位小伙伴分工合作，基本完工

## sharing meeting

近两周举办了2场分享会

- 《Arbitrum retryable ticket》by JasonWan 
  - https://www.youtube.com/watch?v=sYo8DXvysJI
  - https://www.youtube.com/watch?v=l_wt3L2k4dc

- 《零知识证明原理讲解&AMA》by 钟林
  - https://www.youtube.com/watch?v=Tg-Z4ESZ8yg 

- 转载 consensys workshop
  - https://www.youtube.com/watch?v=3cqZWGGHtqs

## 翻译小组

- 正式发布15-16集
  - https://mp.weixin.qq.com/s/cnPLG4deKrWVNCZA6bj9oA
